scarb/src/core/config.rs

```diff
@@ -64,6 +64,10 @@ impl Config {
         &self.manifest_path
     }
 
+    pub fn manifest_dir(&self) -> &Utf8Path {
+        self.manifest_path.parent().unwrap()
+    }
```

Comment:
1. This duplicates `root` :)
2. And this is ambiguous, because with `manifest_path` you are able to use non-standard-named manifest file, for example `Scarb.dev.toml`